src/System.Data.Common/tests/System/Data/Common/RowUpdatedEventArgsTest.cs

```diff
@@ -0,0 +1,71 @@
+﻿using Xunit;
+
+namespace System.Data.Common
+{
+    public class RowUpdatedEventArgsTest
+    {
+        [Fact]
+        public void ConstructorFail()
+        {
+            ArgumentOutOfRangeException ex = Assert.Throws<ArgumentOutOfRangeException>(() => new RowUpdatedEventArgs(null, null, (StatementType)100, null));
+            Assert.Equal(typeof(ArgumentOutOfRangeException), ex.GetType());
+            Assert.Null(ex.InnerException);
+            Assert.True(ex.Message.IndexOf(nameof(StatementType)) != -1);
+            Assert.True(ex.Message.IndexOf("100") != -1);
+            Assert.Equal(ex.ParamName, nameof(StatementType));
+        }
+
+        [Fact]
+        public void ConstructorPass()
+        {
+            var ev = new RowUpdatedEventArgs(null, null, 0, null);
+            Assert.NotNull(ev);
+            Assert.Null(ev.Row);
+            Assert.Null(ev.Command);
+            Assert.Null(ev.TableMapping);
+            Assert.Equal(ev.StatementType, (StatementType)0);
+            Assert.Equal(ev.RowCount, 0);
+            Assert.Equal(ev.RecordsAffected, 0);
+
+            var table = new DataTable();
+            var ev2 = new RowUpdatedEventArgs(table.NewRow(), null, StatementType.Update, new DataTableMapping());
+            Assert.NotNull(ev2);
+            Assert.NotNull(ev2.Row);
+            Assert.Equal(ev2.Row.Table, table);
+            Assert.Equal(ev2.RowCount, 1);
+            Assert.Null(ev2.Command);
+            Assert.Equal(ev2.StatementType, StatementType.Update);
+            Assert.NotNull(ev2.TableMapping);
+        }
+
+        [Fact]
+        public void Status_set()
```

Comment:
Similar to my previous naming comment, this would be better as something like `Status_SetInvalidValue_ThrowsArgumentOutOfRangeException`.